content/id/_index.html

```diff
@@ -44,12 +44,12 @@ <h2>Tantangan yang Dihadapi untuk Melakukan Migrasi 150+ Microservices ke Kubern
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">Hadiri KubeCon North America pada 24-28 Oktober 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Hadiri KubeCon Europe pada 19-22 Maret 2024</a>
```

Comment:
@Arhell Should it be KubeCon + CloudNativeCon? as in upstream EN

Reply:
Done, thanks